SADXModManager/LoaderInfo.cs

```diff
@@ -58,21 +58,32 @@ class SADXLoaderInfo : LoaderInfo
 
 		public bool DisableBASS { get; set; }
 
-        [DefaultValue(-1)] public int TestSpawnLevel { get; set; } = -1;
-        [DefaultValue(0)] public int TestSpawnAct { get; set; } = 0;
+        [DefaultValue(-1)]
+		public int TestSpawnLevel { get; set; } = -1;
+        [DefaultValue(0)]
+		public int TestSpawnAct { get; set; } = 0;
 
-        [DefaultValue(-1)] public int TestSpawnCharacter { get; set; } = -1;
+        [DefaultValue(-1)]
+		public int TestSpawnCharacter { get; set; } = -1;
 
-        [DefaultValue(false)] public bool TestSpawnPositionEnabled { get; set; } = false;
-        [DefaultValue(false)] public bool TestSpawnRotationHex { get; set; } = false;
-        [DefaultValue(0)] public int TestSpawnX { get; set; } = 0;
-        [DefaultValue(0)] public int TestSpawnY { get; set; } = 0;
-        [DefaultValue(0)] public int TestSpawnZ { get; set; } = 0;
-        [DefaultValue(0)] public int TestSpawnRotation { get; set; } = 0;
+        [DefaultValue(false)]
+		public bool TestSpawnPositionEnabled { get; set; } = false;
+        [DefaultValue(false)]
+		public bool TestSpawnRotationHex { get; set; } = false;
+        [DefaultValue(0)]
+		public int TestSpawnX { get; set; } = 0;
+        [DefaultValue(0)]
+		public int TestSpawnY { get; set; } = 0;
+        [DefaultValue(0)]
+		public int TestSpawnZ { get; set; } = 0;
+        [DefaultValue(0)]
+		public int TestSpawnRotation { get; set; } = 0;
 
-        [DefaultValue(-1)] public int TestSpawnEvent { get; set; } = -1;
+        [DefaultValue(-1)]
+		public int TestSpawnEvent { get; set; } = -1;
 
-        [DefaultValue(-1)] public int TestSpawnSaveID { get; set; } = -1;
+        [DefaultValue(-1)]
+		public int TestSpawnSaveID { get; set; } = -1;
```

Comment:
Formatting here is totally borked. Mixed tabs & spaces for indentation. Granted, there's more of this going on in this file already, but we should avoid adding more inconsistency.

Reply:
> I feel like including d3d8to9 is probably out of scope here, though.

I think MainMemory told PkR they could do it. I wonder why it wasn't directly pushed to the main branch though... maybe because we were actively working on this branch, I don't know ^^

Will look at formatting tomorrow, thanks for looking at it